### PR TITLE
Fat 1625 c 196769

### PR DESCRIPTION
**Pull request by TetianaParanich**
## Purpose
- Implement e-2-e automation of test case C9200: Item status date updates

## Refs
- https://foliotest.testrail.io/index.php?/cases/view/9200

### CODE REVIEW

File: cypress/support/api/tags.js [hunk 1]
```diff
@@ -1,7 +1,29 @@
+import uuid from 'uuid';
```
Comment (IhorBohdan) [inline cypress/support/api/tags.js:1]: Delete never used methods

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 2]
```diff
@@ -14,6 +15,11 @@ import {
   Checkbox,
   MultiColumnListRow,
   Link,
+  MultiSelect,
+  Pane,
+  MultiSelectMenu,
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/inventoryInstance.js:20]: Delete never used interactors

File: cypress/support/fragments/data_import/dataImport.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ const uploadFile = (filePathName, fileName) => {
 const wailtLoading = () => {
   cy.expect(Section({ id: 'pane-jobs-title' }).exists());
   cy.expect(Section({ id: 'pane-jobs-title' }).find(HTML(including('Loading'))).absent());
-  cy.expect(Section({ id: 'pane-logs-title' }).find(Button('View all')).exists());
+  cy.expect(Section({ id: 'pane-logs-title' }).find(Button('Actions')).exists());
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/data_import/dataImport.js:16]: Move repeating elements in constant

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 4]
```diff
@@ -233,5 +240,32 @@ export default {
...
+  },
+
+  addTag:(tagName) => {
+    cy.intercept('/tags?limit=10000').as('getTags');
+    cy.do(tagButton.click());
+    cy.wait(['@getTags']);
+    // TODO: clarify with developers what should be waited
+    cy.wait(1000);
+    cy.do(tagsPane.find(TextField({ id:'input-tag-input' })).fillIn(tagName));
+    cy.do(tagsPane.find(TextField({ id:'input-tag-input' })).click());
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/inventory/inventoryInstance.js:252]: Move repeating elements in constant

File: cypress/support/fragments/inventory/inventoryInstances.js [hunk 5]
```diff
@@ -21,5 +31,19 @@ export default {
...
+  resetAllFilters:() => {
+    cy.do(Pane('Search & filter').find(Button('Reset all')).click());
+  },
+
+  searchByTag:(tagName) => {
+    cy.do(Button({ id:'accordion-toggle-button-instancesTags' }).click());
+    // wait for data to be loaded
+    cy.intercept('/search/instances/facets?facet=instanceTags**').as('getTags');
+    cy.do(Section({ id:'instancesTags' }).find(TextField()).click());
+    cy.do(Section({ id:'instancesTags' }).find(TextField()).fillIn(tagName));
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/inventory/inventoryInstances.js:45]: Move repeating elements in constant

File: cypress/support/fragments/inventory/inventorySearch.js [hunk 6]
```diff
@@ -165,7 +165,7 @@ export default {
     });
   },
 
-  searchByParameter: (parameter, value) => {
+  searchInstanceByParameter: (parameter, value) => {
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/inventory/inventorySearch.js:168]: Are you sure this name changing will not affect other tests?

File: cypress/integration/inventory/assing-tags-to-an-instance-record.spec.js [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+
+describe('ui-inventory: Assign tags to an Instance record', () => {
+  const instanceTitle = `autoTestInstanceTitle.${getRandomPostfix()}`;
+  const tag = {
+    id: uuid(),
+    description: 'autotag',
+    label: 'autotag'
```
Comment (MaksimTerekhov) [inline cypress/integration/inventory/assing-tags-to-an-instance-record.spec.js:14]: Leave a comment please why you don't use generate random string
If you have a jira issue with question to PO about tag length limit, leave it here pls

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 8]
```diff
@@ -21,6 +25,7 @@ import InventoryViewSource from './inventoryViewSource';
 import NewHoldingsRecord from './newHoldingsRecord';
 import InventoryInstanceSelectInstanceModal from './holdingsMove/inventoryInstanceSelectInstanceModal';
 import InventoryInstancesMovement from './holdingsMove/inventoryInstancesMovement';
+import InventoryInstances from './inventoryInstances';
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/inventory/inventoryInstance.js:28]: 'InventoryInstances' is defined but never used.
Please check it

**Pull request merged**